static/_redirects

```diff
@@ -369,6 +369,10 @@
 /docs/support   /docs/guides/database/troubleshooting-orm
 /docs/support/*   /docs/guides/database/troubleshooting-orm/:splat 301!
 
+## Moving PDP as a top-level section
+
+/docs/concepts/data-platform     /docs/data-platform
```

Comment:
I think the first line might be missing the `301!`.
Line 369 probably also has the same problem (where I guess this is modelled after)

Reply:
Right. I've added `301!` to both now.